pydantic_ai_slim/pydantic_ai/messages.py

```diff
@@ -166,6 +187,7 @@ def __init__(
         vendor_metadata: dict[str, Any] | None = None,
         media_type: str | None = None,
         kind: Literal['video-url'] = 'video-url',
+        identifier: str | None = None,
         *,
```

Comment:
Please move the `*` ahead of `force_download` so all arguments other than `url` need to be keywords arguments -- same for the other subclasses